packages/mco/constants/disaster-recovery.ts

```diff
@@ -9,6 +9,8 @@ export const DRPC_NAMESPACE_ANNOTATION =
 // DRPC annotations
 export const LAST_APP_DEPLOYMENT_CLUSTER_ANNOTATION =
   'drplacementcontrol.ramendr.openshift.io/last-app-deployment-cluster';
+export const APP_NAMESPACE_ANNOTATION =
+  'drplacementcontrol.ramendr.openshift.io/app-namespace'; // ACM managed application workload namespace
```

Comment:
nits:
```suggestion
// ACM managed application workload namespace
export const APP_NAMESPACE_ANNOTATION =
  'drplacementcontrol.ramendr.openshift.io/app-namespace';
```
not really needed, up to you if you want to make this minor change...

Reply:
let it be, if ramen changes it is easy to update  here.

Reply:
suggestion was to shift the inline comment on top, separate line :) (not related to ramen)

Reply:
ack